client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -11,8 +10,7 @@ import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import schema from './schema.json';
 
 const noop = () => {};
-const isRunningInJetpackSite = config.isEnabled( 'is_running_in_jetpack_site' );
-const jitmSchema = isRunningInJetpackSite ? { ...schema, ...schema.properties.data } : schema;
+const jitmSchema = schema;
```

Comment:
In `transformApiRequest` there is a bit of code that handles the case when `jitms` has the `data` field. That can be now also removed.

Reply:
Fixed